docs/UsersGuide/RELEASE.txt

```diff
@@ -1,43 +1,68 @@
 The current HDFView products can be downloaded at:
-        https://www.hdfgroup.org/downloads/hdfview/
+        https://github.com/HDFGroup/hdfview/releases
 
 
 HDF Libraries
 =============================================================================
-This release was built and tested with HDF 4.2.15 and HDF5 1.10.7.
-
-Windows ONLY: HDFView built with VS 2015 will require that
-the Visual C++ Redistributable Packages be installed on your machine.
-If you cannot access HDF5 files then you do not have these packages.
-You can obtain them from:
-         https://www.microsoft.com/en-us/download/details.aspx?id=48145
+This release was built and tested with HDF 4.3.x and HDF5 1.15.x.
 
 
 JDK
 =============================================================================
-This release was built and tested with OpenJDK 15. It uses modules and the
-newest jpackage for distribution.
+This release was built and tested with OpenJDK 21. It uses modules, the
+newest jpackage for distribution and adds support for float16 datatypes.
+
 
 Major Enhancements
 ==============================================================================
-    *
+    * GH #117 Add User Option for editing the plugin path and include plugins.
+    * GH #138 Add Support for operating with float16 datatypes.
+
+    ***** Previous fixes *****
+    * GH #121 Updated all icons with transparency.
 
 Major Bug Fixes
 ==============================================================================
-    * Added support to read and display non-standard floating point numbers.
-    * Added optional startDir to command line options.
-    * Corrected user.home vs user.dir confusion.
-    * Split Current Working Directory options into three selections:
-      a. Current Work Directory or user.dir system property
-      b. User Home Directory or user.home system property
-      c. User selected directory
+    *
+
+    ***** Previous fixes *****
+    * GH #119 (crashes opening HDF4 file) has been fixed. The problem was
+      that the HDF4 xdr.dll file was not copied from the hdf4 install location.
+
+    * HDFVIEW-284 (crashes on NETCDF-4 grids) has been fixed. The problem was
+      that references in variable-length containers were handled like strings.
```

Comment:
These 2 sentences seem to say the same thing

Reply:
The actual file should be a duplicate of the main RELEASE.txt file. Actually, this should be a link/reference back to the main file.